src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,6 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return 'the responses are identical';
```

Comment:
would it not be "are *not* identical"? and could we use a message transformer here to give more hints what is going wrong?